app/layout.tsx

```diff
@@ -1,23 +1,33 @@
-/** @format */
+"use client";
 
-import type { Metadata } from "next";
 import { Inter } from "next/font/google";
 import "./globals.css";
 import { cn } from "../lib/utils";
 import SideNavbar from "@/components/SideNavbar";
+import { useEffect, useState } from "react";
+import { isLoginPage } from "@/lib/utils";
 
 const inter = Inter({ subsets: ["latin"] });
 
-export const metadata: Metadata = {
```

Comment:
Why did you remove that? Please add again

Reply:
It was removed to fix issue of adding custom sidebar that should not be visible on login page. So layout was made a client side component.

We can add meta data using <head> tag within the layout.